scripts/prepdocslib/searchmanager.py

```diff
@@ -131,19 +168,25 @@ async def create_index(self):
                         VectorSearchProfile(
                             name="embedding_config",
                             algorithm="hnsw_config",
+                            vectorizer="myOpenAI" if self.use_int_vectorization else None,
```

Comment:
It won't work with `azure-search-documents==11.4.0b11`
suggest changing to
`vectorizer=f"{self.search_info.index_name}-vectorizer" if self.use_int_vectorization else None,`

Reply:
Commit was ahead to due PR comments. Thanks for catching. Resolved it.